cmd/sales-api/handlers/user.go

```diff
@@ -13,9 +14,17 @@ import (
 	"go.opencensus.io/trace"
 )
 
+// UserAuth knows everything needed to authenticate Users.
```

Comment:
This comment was a bit confusing to me ... authenticating a user could refer to two things: authenticating for getting a token or authenticating via validating a token (on another endpoint)

Reply:
We haven't yet wired up the middleware for validating tokens but I believe this type is how we will pass the public key down to the API so it will eventually mean both kinds of authenticate.